ran-simulator/values.yaml

```diff
@@ -5,6 +5,7 @@
 replicaCount: 1
 
 image:
+  depCheck: quay.io/stackanetes/kubernetes-entrypoint:v0.3.1
```

Comment:
What's this for - never saw it before?

Reply:
It is the dependency checker by reading Kubernetes metadata and manifest. With it, we can handle dependencies. For example, HO application should be deployed after ONOS-RIC is ready. That image is running as a InitContainer. First, that initContainer reads K8s metadata and manifest and then check whether ONOS-RIC container is ready. If it is ready, initContainer is finished and then go to run main container(s) like onos-ric-ho container. Otherwise, initContainer waits until onos-ric is ready.

Reply:
You can find it in initContainer section in deployments.yaml file.